platform-configs/include/msi-z690-common.robot

```diff
@@ -86,6 +90,7 @@ ${MEMORY_PROFILE_SUPPORT}=                      ${TRUE}
 ${ESP_SCANNING_SUPPORT}=                        ${TRUE}
 ${DCU_UUID_SUPPORT}=                            ${TRUE}
 ${DCU_SERIAL_SUPPORT}=                          ${TRUE}
+${RESET_TO_DEFAULTS_SUPPORT}=                   ${TRUE}
```

Comment:
That was disabled mostly because of the problems with serial being disabled after resetting. We can rather quickly test these manually after we finish automated tests.

Reply:
see: https://github.com/Dasharo/open-source-firmware-validation/pull/530/files#r1806136564

Reply:
This is a part of: https://github.com/Dasharo/open-source-firmware-validation/issues/540